File: Selection-sort/c/SelectionSort.c
#include <stdio.h>

void selectionSort(int arr[], int n)
{
    int min;
    int temp;
    for(int i = 0; i < n-1; j++)
    {
        int min = i;
        for(int j = i + 1; j < n;)
    }

}

int main() 
{
    int arr[] = {74, 34, 25, 12, 22, 11, 90};
    int length = sizeof(arr) / sizeof(arr[0]);
    printf("Unsorted array:\n");
    for (int i = 0; i < length - 1; i++) 
    {
        printf("%d ", arr[i]);
    }
    printf("\n");
    selectionSort(arr, length);
    printf("Sorted array:\n");
    for (int i = 0; i < length - 1; i++) 
    {
        printf("%d ", arr[i]);
    }
    printf("\n");
    return 0;
}

File: Selection-sort/python/SelectionSort.py
def selectionSort(arr, n):
    for i in range(n-1):
        min = i
        j = i + 1
        for j in range(i+1, n):
            if (arr[j] < arr[min]):
                min = j
        temp = arr[min]
        arr[min] = arr[i]
        arr[i] = temp
                

arr = [64, 34, 25, 12, 22, 11, 90]
print("Unsorted array:")
for i in range(len(arr)):
    print(arr[i], end=" ")
print()
selectionSort(arr, len(arr))
print("Sorted array:")
for i in range(len(arr)):
    print(arr[i], end=" ")
print()